docs/dev/Lua API.rst

```diff
@@ -5130,6 +5130,17 @@ widget does not require direct usage of ``Tab``.
            usage of ``Tab`` in ``TabBar:init()`` for an example. See the default value of ``active_tab_pens`` or ``inactive_tab_pens``
            in ``TabBar`` for an example of how to construct pens.
 
+RangeSlider class
+-----------------
+
+This widget implements a mouse-interactable range-slider. The user can move its two handles to set minimum and maximum values to define a range.
```

Comment:
Should also mention that both handles can be moved at once by dragging the bar between the two handles.

Reply:
wow. i never realized this was a thing, and would've never expected it to be a thing 😆 
i mean, never in my life i had encountered a range slider in which you could _drag the line to move the handles_ 🤯 

Was about to say that the handles really need to provide some feedback on hover, like the text turning white or something... but man. The line would _also_ need to provide such feedback! Nobody is ever going to guess that this functionality exists! Should it even exist?

i mean, having one slider "push" the other when they're both on the same value is intuitive. This is just... i don't know, we maybe need to re-work the graphics to make this more obvious. :\ 
Instead of a green line, it should become an extension of the yellow handles, or something.